tests/nn/data_parallel/test_fsdp_offload.py

```diff
@@ -322,28 +324,37 @@ def _test_ssd_offloading_train_flatten_params_wrapper(self, rank, group, config)
                     input = model.get_input(torch.device("cuda"))
                     output = model(*input)
                     pre_checkpoint_last_output = output
+                    """
```

Comment:
remove?

Reply:
These were very helpful to debugging the tests when everything ran to completion, but the outputs from the pre-checkpoint and post-checkpoint models didn't match. I think they should stay if that's permissible.